app/src/main/kotlin/com/mapzen/erasermap/presenter/MainPresenterImpl.kt

```diff
@@ -61,6 +64,7 @@ public open class MainPresenterImpl(val mapzenLocation: MapzenLocation, val bus:
     }
 
     override fun onReverseGeocodeResultsAvailable(searchResults: Result?) {
+        reverseGeo = true
```

Comment:
Not sure if `reverseGeo = true` is required here, since its already set before `onReverseGeocodeResultsAvailable` gets called on [line 411](https://github.com/mapzen/eraser-map/blob/438-clear-mapdata/app/src/main/kotlin/com/mapzen/erasermap/presenter/MainPresenterImpl.kt#L411) and [line422](https://github.com/mapzen/eraser-map/blob/438-clear-mapdata/app/src/main/kotlin/com/mapzen/erasermap/presenter/MainPresenterImpl.kt#L422).